src/Pages/wonkeunchoi/Login/Login.scss

```
@@ -0,0 +1,92 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+%flex-center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.Login_wonkeunchoi {
+  @extend %flex-center;
+  flex-direction: column;
+  margin: 0 auto;
+  height: 100vh;
+
+  article {
+    padding-top: 10px;
+    padding-bottom: 10px;
+
+    .westagram {
+      @extend %flex-center;
+      flex-direction: column;
+      width: 378px;
+      height: 450px;
+      border: 1px solid #d3d3d3;
+      border-radius: 1px;
+      box-shadow: 3px 4px 2px 10px #f5f5f5;
+
+      h1 {
+        display: flex;
+        align-items: flex-start;
+        margin: 24px auto 0 auto;
+        font-size: 40px;
+        font-family: 'Lobster', cursive;
+      }
+
+      section {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        width: 378px;
+        height: 270px;
+        margin-bottom: 70px;
+
+        #loginForm {
+          display: flex;
+          flex-direction: column;
+          align-items: center;
+          width: 100%;
+          line-height: 12px;
+
+          input {
+            width: 300px;
+            height: 44px;
+            margin-bottom: 12px;
+            padding: 0 12px;
+            border: 2px solid #f5f5f5;
+            border-radius: 3px;
+            background-color: #f6f6f8;
+            font: inherit;
+          }
+
+          #btn_login {
+            width: 80%;
+            height: 35px;
+            margin-top: 15px;
+            font-style: inherit;
+            font-weight: bold;
+            color: white;
+            background-color: #458eff;
+            border: 0px solid #458eff;
+            border-radius: 5px;
+          }
+
+          #btn_login:disabled {
+            opacity: 0.3;
+          }
```

```suggestion
          #btn_login {
            width: 80%;
            height: 35px;
            margin-top: 15px;
            font-style: inherit;
            font-weight: bold;
            color: white;
            background-color: #458eff;
            border: 0px solid #458eff;
            border-radius: 5px;

          &:disabled {
            opacity: 0.3;
          }
```
- 앞서 드린 코멘트와 동일한 이유로 id 셀렉터는 피하는 것이 좋습니다
- & 연산자를 사용해 nesting 가능할 것 같습니다